week2/intro_to_property_packages.ipynb

```diff
@@ -0,0 +1,430 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "33f1fceb",
+   "metadata": {},
+   "source": [
+    "# Property Package Introduction\n",
+    "\n",
+    "The properties of the water in WaterTAP models is defined by the [property models](https://watertap.readthedocs.io/en/stable/technical_reference/property_models/index.html). \n",
+    "\n",
+    "In unit models, properties exist as sub-models of the main unit model on what are called _state blocks_. \n",
```

Comment:
I can see this statement being confusing to new people... not exactly sure how to simplify it tho

Reply:
Agreed. I think for the "verbose" version, we should just explain StateBlocks in terms that a practitioner would better understand--they represent stream specifications and describe physico-chemical properties of a given stream.